packages/backend/src/apps/m365-excel/actions/update-table-row/index.ts

```diff
@@ -0,0 +1,169 @@
+import type { IRawAction } from '@plumber/types'
+
+import z from 'zod'
+
+import StepError from '@/errors/step'
+
+import {
+  constructMsGraphValuesArrayForRowWrite,
+  convertRowToHexEncodedRowRecord,
+} from '../../common/workbook-helpers/tables'
+import WorkbookSession from '../../common/workbook-session'
+import getTableRowAction from '../get-table-row'
+import getTableRowImpl from '../get-table-row/implementation'
+
+import getDataOutMetadata from './get-data-out-metadata'
+import {
+  dataOutSchema,
+  parametersSchema,
+  updateRowValuesResponseSchema,
+} from './schemas'
+
+type DataOut = Required<z.infer<typeof dataOutSchema>>
+
+const action: IRawAction = {
+  name: 'Update table row',
+  key: 'updateTableRow',
+  description: 'Updates a single row of data in your Excel table',
```

Comment:
@pregnantboy what do you think of `Updates the first matching row of data in your Excel table`.

Reply:
Have updated copy in getTableRow action instead. Pull and lmk what you think